nautobot/extras/tasks.py

```diff
@@ -14,7 +14,7 @@
 
 
 @nautobot_task
-def update_custom_field_choice_data(field_id, old_value, new_value):
+def update_custom_field_choice_data(field_id, old_value, new_value, change_context={}):
```

Comment:
Don't use mutable objects as defaults. :)

```suggestion
def update_custom_field_choice_data(field_id, old_value, new_value, change_context=None):
```